registry/recipes/nginx.md

```diff
@@ -38,11 +38,10 @@ you want through the secondary authentication mechanism implemented inside your
 proxy, it also requires that you move TLS termination from the Registry to the
 proxy itself.
 
-> Another important thing to note is that by binding your registry to
-> `localhost:5000` without authentication, you open up a potential loophole in
-> your Docker Registry security - anyone who can log on to the server where your
-> Docker Registry is running can push images to your registry, without
-> authentication. This could have potentially devastating effects.
+> ***NOTE:*** Docker does not recommend binding your registry to `localhost:5000` without
+> authentication. This creates a potential loophole in your Docker Registry security.
+> As a result, anyone with access to your Docker Registry can push images without
```

Comment:
s/anyone with access to your Docker Registry/anyone who can log on to the server where your Docker Registry is running/

Reply:
resolved